### PR TITLE
Test pulp in rerouted environment

### LINKED ISSUE #578: Add tests to run pulp in a rerouted environment
For now, we will just reroute the container in the s3 test environment.

### PR DESCRIPTION
**Pull request by mdellweg**
fixes #578

### CODE REVIEW

File: templates/github/.github/workflows/scripts/install.sh.j2 [hunk 1]
```diff
@@ -150,6 +150,9 @@ if [ "$TEST" = "s3" ]; then
   sed -i -e '$a s3_test: true\
 minio_access_key: "'$MINIO_ACCESS_KEY'"\
 minio_secret_key: "'$MINIO_SECRET_KEY'"' vars/main.yaml
+
+  echo "PULP_API_ROOT=/rerouted/djnd/" >> "$GITHUB_ENV"
+  export PULP_API_ROOT="/rerouted/djnd/"
```
Comment (mdellweg) [inline templates/github/.github/workflows/scripts/install.sh.j2:155]: This should really be the lift of that feature. Do we want to set the vars in a different way?

cc @fao89

**Comment by mdellweg** [comment]
Results here:

https://github.com/pulp/pulp_file/pull/644

File: templates/github/.github/workflows/scripts/install.sh.j2 (hunk 1 above)
Comment (fao89) [reply templates/github/.github/workflows/scripts/install.sh.j2:155]: I think this is good

**Review by fao89 — APPROVED** [review]

**Comment by mdellweg** [comment]
How do we want to handle failing tests once this gets rolled out to plugins?

**Comment by fao89** [comment]
> How do we want to handle failing tests once this gets rolled out to plugins?

I would trigger the update job and fix the tests on plugins that break

**Comment by mdellweg** [comment]
> > How do we want to handle failing tests once this gets rolled out to plugins?
> 
> I would trigger the update job and fix the tests on plugins that break

I'd take at least core and file. (Have them already.)

With that i think we are good to go.

**Pull request merged**